src/event.rs

```diff
@@ -33,6 +33,17 @@ pub enum NodeElderChange {
     None,
 }
 
+/// Bound name of elders and section_key, section_prefix info together.
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
This is a `SectionAuthorityProvider` here. So this struct can go along with SectionInfo AFAIK